recipes/mpdecimal/2.5.x/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup(TARGETS)
+
+# Workaround for -rpath on arm64: https://cmake.org/cmake/help/latest/release/3.20.html#id2
```

Comment:
There's a fix apple name hepler in conan for this AFAIK 🤔

Reply:
Looks like this is fixed now?